src/internal/packager/helm/chart.go

```diff
@@ -139,12 +139,20 @@ func (h *Helm) InstallOrUpgradeChart() (types.ConnectStrings, string, error) {
 }
 
 // TemplateChart generates a helm template from a given chart.
-func (h *Helm) TemplateChart() (string, chartutil.Values, error) {
+func (h *Helm) TemplateChart() (manifest string, chartValues chartutil.Values, err error) {
 	message.Debugf("helm.TemplateChart()")
 	spinner := message.NewProgressSpinner("Templating helm chart %s", h.chart.Name)
-	defer spinner.Stop()
+	defer func() {
+		if err != nil {
+			spinner.Stop()
+			spinner.Errorf(err, "Problem rendering the helm template for %s: %s", h.chart.Name, err.Error())
+		} else {
+			spinner.Success()
+		}
```

Comment:
why have this complex logic?

Reply:
The spinner wasn't stopping on errors. I could have put spinner.stop on each of the error conditions, but I thought this might be cleaner.

Reply:
I believe I was playing around with the spinner back when I first worked on this. I needed to do a more diligent self review on this one